spec/components/trainees/confirmation/lead_school/view_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Trainees
+  module Confirmation
+    module LeadSchool
+      describe View do
+        alias_method :component, :page
+
+        let(:school) { create(:school, open_date: Time.zone.today) }
+        let(:trainee) { create(:trainee, lead_school: school) }
+
+        before do
+          render_inline(View.new(data_model: trainee))
+        end
+
+        it "renders the lead school" do
+          expect(component.find(".govuk-summary-list__row.lead-school .govuk-summary-list__value"))
```

Comment:
you could probably wrap `component.find(".govuk-summary-list__row.lead-school .govuk-summary-list__value")` into a `subject { component.find(".govuk-summary-list__row.lead-school .govuk-summary-list__value") }` then do `expect(subject).to have_text(...)` to clean these up a bit